cql2pgjson-cli/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.folio</groupId>
     <artifactId>raml-module-builder</artifactId>
-    <version>34.1.0-SNAPSHOT</version>
+    <version>34.2.0-SNAPSHOT</version>
```

Comment:
Why not RMB 35.0.0-SNAPSHOT? You argued that RMB-927 is a breaking change (I don't see how, but still).